src/app/tests/suites/TestBinding.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Binding Tests
+
+config:
+    nodeId: 0x12344321
+    cluster: "Binding"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Write empty binding table"
+      command: "writeAttribute"
+      attribute: "binding"
+      arguments:
+          value: []
+
+    - label: "Read empty binding table"
+      command: "readAttribute"
+      attribute: "binding"
+      response:
+          value: []
+
+    - label: "Write invalid binding table"
+      command: "writeAttribute"
+      attribute: "binding"
+      arguments:
+          value:
+              [
+                  { FabricIndex: 1 },
```

Comment:
We should probably just use `FabricIndex: 0` for all these fabric indices.  It gets ignored on the other side anyway, and we are likely to do something smart for `FabricIndex: 0` in terms of not encoding it.